src/features/device-heartbeat/index.ts

```diff
@@ -150,6 +151,8 @@ interface MetricEventArgs {
 interface DeviceOnlineStateManagerMessage {
 	id: string;
 	currentState: DeviceOnlineStates;
+	/** Cache Busting value */
```

Comment:
I know it stands next to it, but cb is very commonly used for callback.

Reply:
TBH I could't figure out what the ideal short prop name would be, so I asked an AI bot what is the most common name used for this use case and it sounded convincing enough to me. Better ideas are welcome of course.
Also, callbacks shouldn't be that common nowadays, right?